retrieval_service/datastore/providers/neo4j_graph.py

```diff
@@ -105,12 +107,41 @@ async def create_amenities(tx, amenities):
                     category=amenity.category,
                 )
 
+            # Create relationships from CSV
+            # Create SIMILAR_TO relationship
+            csv_file_path = "../data/relationships/amenity_relationships.csv"
+
+            with open(csv_file_path, "r") as file:
+                reader = csv.DictReader(file)
+                for row in reader:
+                    src_name = row["src_id"]
+                    rel_type = row["rel_type"]
+                    tgt_name = row["tgt_id"]
+
+                    # Generate and run the Cypher query
+                    # Case-insensitive and apostrophes-insensitive match
+                    result = await tx.run(
+                        f"""
+                        MATCH (a:Amenity) WHERE toLower(a.name) = toLower("{src_name}")
```

Comment:
Graph builder node name output looks like this:

`Hawaii coffee store`

And the node name in the graph (created using the names on the dataset file) looks like this:

`Hawaii Coffee Store`

Since the output in the graph builder is not entirely lower-case I have to turn both into lowercase for comparison